File: CS/02_Text/DrawTextWithGradient.cs
using Spire.Pdf;
using Spire.Pdf.Graphics;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace DrawTextWithGradient
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a pdf document
            PdfDocument doc = new PdfDocument();

            //Add a new page
            PdfPageBase page = doc.Pages.Add();

            //Create a rectangle
            Rectangle rect = new Rectangle(new Point(0, 0), new Size(300, 100));

            //Create a brush with gradient
            PdfLinearGradientBrush brush = new PdfLinearGradientBrush(rect, Color.Red, Color.Blue, PdfLinearGradientMode.Horizontal);

            //Create a true type font with size 20f, underline style
            PdfTrueTypeFont font = new PdfTrueTypeFont(new Font("Arial", 20, FontStyle.Underline));
           
            //Draw text
            page.Canvas.DrawString("Welcome to E-iceblue!", font, brush, new Point(0, 100));

            String result="DrawWithGradient-result.pdf";
            //Save to file
            doc.SaveToFile(result);
            
            //Launch the Pdf file
            PDFDocumentViewer(result);
        }
        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }  
    }
}

File: VB/02_Text/DrawTextWithGradient.vb
Imports Spire.Pdf
Imports Spire.Pdf.Graphics

Namespace DrawTextWithGradient
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new PdfDocument object
			Dim doc As New PdfDocument()

			' Add a new page to the document
			Dim page As PdfPageBase = doc.Pages.Add()

			' Define a rectangle with a specified position and size
			Dim rect As New Rectangle(New Point(0, 0), New Size(300, 100))

			' Create a linear gradient brush with a specified rectangle and colors
			Dim brush As New PdfLinearGradientBrush(rect, Color.Red, Color.Blue, PdfLinearGradientMode.Horizontal)

			' Create a TrueType font with Arial font family, size 20, and underline style
			Dim font As New PdfTrueTypeFont(New Font("Arial", 20, FontStyle.Underline))

			' Draw a string on the page's canvas using the specified font, brush, and position
			page.Canvas.DrawString("Welcome to E-iceblue!", font, brush, New Point(0, 100))

			' Specify the file name for saving the document
			Dim result As String = "DrawWithGradient-result.pdf"

			' Save the document to the specified file
			doc.SaveToFile(result)

			' Close the document
			doc.Close()

			' Launch the Pdf file
			PDFDocumentViewer(result)
		End Sub
		Private Sub PDFDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
